src/main/java/subway/line/service/LineService.java

```diff
@@ -0,0 +1,79 @@
+package subway.line.service;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import subway.line.dto.LineRequest;
+import subway.line.dto.LineResponse;
+import subway.line.dto.LineUpdateRequest;
+import subway.line.entity.Line;
+import subway.line.repository.LineRepository;
+import subway.station.entity.Station;
+import subway.station.repository.StationRepository;
+
+import javax.persistence.EntityNotFoundException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Transactional(readOnly = true)
+public class LineService {
+
+    private final LineRepository lineRepository;
+    private final StationRepository stationRepository;
+
+    public LineService(LineRepository lineRepository, StationRepository stationRepository) {
+        this.lineRepository = lineRepository;
+        this.stationRepository = stationRepository;
+    }
+
+    @Transactional
+    public LineResponse createSubwayLine(final LineRequest request) {
+        final String lineName = request.getName();
+        final String lineColor = request.getColor();
+        final Station upStation = this.findStationById(request.getUpStationId());
+        final Station downStation = this.findStationById(request.getDownStationId());
+        final Integer lineDistance = request.getDistance();
+
+        final Line newLine = new Line(lineName, lineColor, upStation, downStation, lineDistance);
+        final Line savedLine = lineRepository.save(newLine);
+
+        return LineResponse.convertToDto(savedLine);
+    }
+
+    public LineResponse getSubwayLine(final Long lindId) {
+        final Line line = this.findLineById(lindId);
+
+        return LineResponse.convertToDto(line);
+    }
+
+    public List<LineResponse> getSubwayLines() {
+        return lineRepository.findAll().stream()
+                .map(line -> LineResponse.convertToDto(line))
```

Comment:
메서드 레퍼런스를 사용해 보셔도 좋을 것 같아요
```suggestion
                .map(LineResponse::convertToDto)
```